tests/approaches/test_nsrt_learning_approach.py

```diff
@@ -154,6 +159,56 @@ def test_nsrt_learning_approach():
                            strips_learner=strips_learner)
 
 
+def test_saving_and_loading_atoms():
```

Comment:
how long does this test take?

Reply:
1.19s. Interestingly, it's not even the most expensive test in this file.